docs/semgrep-ci/overview.md

```diff
@@ -71,7 +71,13 @@ To add Semgrep CI to any CI environment, use the [`returntocorp/semgrep`](https:
 For full project scans:
 
 ```sh
-semgrep scan --config auto --config <other rule or rulesets>
+semgrep ci
+```
+
+For diff-aware scans:
+
+```sh
+SEMGREP_BASELINE_REF=<ref> semgrep ci
```

Comment:
```suggestion
<pre class="language-bash">
<code>SEMGREP_BASELINE_REF=<span classname="placeholder">REF</span></code>
semgrep ci
</pre>
```
We use a specific "placeholder" format for variables users should change ("ref" in this case). Just making it match our conventions.